STEERING.md

```diff
@@ -0,0 +1,51 @@
+# Linkerd Steering Committee Charter
+
+The goal of the Linkerd Steering Committee is to ensure that Linkerd meets the
+needs of its current and future users. The Steering Committee represents the
+voice of the user, and works with the Linkerd maintainers to help guide
+development efforts towards solving concrete and immediate problems for Linkerd
+adopters.
+
+# Responsibilities
+
+The Steering Committee’s responsibilities are to:
+
+1. Provide feedback to project maintainers about Linkerd's featureset, UX, and
+   operational model.
+2. Assist Linkerd maintainers in prioritizing upcoming roadmap items and
+   planned work.
+3. Represent the "voice of the Linkerd user" both internally and to the broader
+   community.
+4. Provide neutral mediation for non-technical disputes.
+5. Develop and maintain a project continuity plan.
+
+# Membership
```

Comment:
Where is the list of members recorded? This doc?

Reply:
I was thinking of a separate doc (STEERING-MEMBERS.md?) for consistency with GOVERNANCE.md and MAINTAINERS.md, but I'm ok with just putting them in here if that's easier.... either way as a separate PR.

Reply:
perhaps a section in MAINTAINERS.md